src/coreComponents/mesh/FaceElementSubRegion.cpp

```diff
@@ -568,27 +597,31 @@ SortedArray< localIndex > makeSortedArrayIota( localIndex newSize, localIndex va
 
 /**
  * @brief Builds the 2d face to 2d elements mapping.
- * @param num2dFaces The number of 2d faces.
- * @param num2dElems The number of 2d elements.
  * @param elem2dToEdges The 2d element (geometrical faces in 3d) to edges mapping.
  * @param edgesTo2dFaces The edges to 2d faces (geometrical edges in 3d) mapping.
```

Comment:
`edgesTo2dFaces` here the edges are the 3D edges right?

Reply:
Yes, every where in the code the quantity is assumed 3d (unless tagged 2d).